packages/dashboard/src/app/styles.css

```
@@ -29,6 +29,14 @@ html[dir="rtl"] * {
  */
 @import 'virtual:admin-theme-inline';
 
+/*
+ * Replaced at build time by the themeVariablesPlugin with the @import
+ * statements for any stylesheets passed via the `theme.additionalStylesheets`
+ * option. Placed here so user styles participate in Tailwind's build pipeline.
+ * When no stylesheets are configured, this placeholder is removed.
+ */
+@import 'virtual:vendure-user-styles';
```

Stylelint (17.13.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)